CElegans/pythonScripts/c302/tune/c302NetTuner.py

```diff
@@ -797,6 +895,1771 @@ def run_optimisation(prefix,
+        target_data['VB3/0/GenericNeuronCell/v' + ':first_spike_time'] = 141
+        target_data['VB4/0/GenericNeuronCell/v' + ':first_spike_time'] = 149
+        target_data['VB5/0/GenericNeuronCell/v' + ':first_spike_time'] = 157
+        target_data['VB6/0/GenericNeuronCell/v' + ':first_spike_time'] = 165
+        target_data['VB7/0/GenericNeuronCell/v' + ':first_spike_time'] = 173
+        target_data['VB8/0/GenericNeuronCell/v' + ':first_spike_time'] = 181
+        target_data['VB9/0/GenericNeuronCell/v' + ':first_spike_time'] = 189
+        target_data['VB10/0/GenericNeuronCell/v' + ':first_spike_time'] = 197
+        target_data['VB11/0/GenericNeuronCell/v' + ':first_spike_time'] = 205
+
+        weights = {}
+        weights['VB1/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB2/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB3/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB4/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB5/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB6/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB7/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB8/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB9/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB10/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+        weights['VB11/0/GenericNeuronCell/v' + ':first_spike_time'] = 10
+
+        input_list = [
+            ('AVBL', '50ms', '600ms', '15pA'),
+            ('AVBR', '50ms', '600ms', '15pA'),
+        ]
+
+        simulator = 'jNeuroML_NEURON'
+        run_optimisation('Delay_AVB_VB_Case2_800_10k_200_200_0.1_20',
+                         'AVB_VB',
+                         'C2',
+                         parameters,
+                         max_constraints,
+                         min_constraints,
+                         weights,
+                         target_data,
+                         config_package='notebooks.configs.AVB',
+                         data_reader='UpdatedSpreadsheetDataReader',
+                         sim_time=700,
+                         dt=0.05,
+                         population_size=800,
+                         max_evaluations=10000,
+                         num_selected=200,
+                         num_offspring=200,
+                         mutation_rate=0.1,
+                         num_elites=20,
+                         seed=12347,
+                         nogui=nogui,
+                         simulator=simulator,
+                         input_list=input_list,
+                         num_local_procesors_to_use=20)
+
+
+    elif '-fw-C2' in sys.argv:
+
+        sim_time = 4000
+
+        parameters = OrderedDict()
+        min_constraints = []
+        max_constraints = []
+
+        parameters['AVBL_to_AVBR_exc_syn_conductance'] = {'default_unit': 'nS'}
```

Comment:
@lungd Are these necessary here? Fine to keep them as long as they're not temp additions... Have you checked to see if some of the other optimisation examples in these (tuning) files still work?